### ISSUE TITLE
System notifications instead of the current "bottom of screen" toast messages

### ISSUE THREAD

**Issue by KairuByte** [Phalcode/gamevault-app#166]
**Is your feature request related to a problem? Please describe.**
GV procedures are often going to take some time. Users are often going to background, minimize or close the UI while things are going on. 

**Describe the solution you'd like**
System level notifications will actually keep the user informed of happenings when the application is not in focus.

**Describe alternatives you've considered**
A custom notification system would work as well.

**Comment by Alfagun74** [comment]
What notifications specifically do you mean?

For instance i think validation errors dont belong there. Because the user has to have the window open to trigger them. But i can certainly understand you want tray notifications for:
- Download Complete
- Download Failed
- Extraction Complete
- Extraction Failed
- etc.

**Comment by KairuByte** [comment]
Yes, those are the type of notification I'm referring to. Validation errors I don't actually think should be toast messages at all, but I should have specified the types.
